.github/workflows/ElastixGitHubActions.yml

```diff
@@ -13,25 +13,25 @@ jobs:
     strategy:
       max-parallel: 3
       matrix:
-        os: [ubuntu-18.04, windows-2019, macos-10.15]
+        os: [macos-10.15, ubuntu-18.04, windows-2019]
         include:
           - os: ubuntu-18.04
             c-compiler: "gcc"
             cxx-compiler: "g++"
-            itk-git-tag: "v5.1.2"
+            itk-git-tag: "0fce3ff86b9242a298a55a42337769eca8e8a4c8"
```

Comment:
@ViktorvdValk Thanks so far... Regarding the upgrade of ITK, I would prefer to have that as a separate pull request. Probably "v.5.2.0", instead of "0fce3ff86b9242a298a55a42337769eca8e8a4c8", right?

Reply:
Ah, I see you adjusted the itk-git-tag to "v5.2.0"  😃